pages/products/[slug].tsx

```diff
@@ -235,10 +268,55 @@ const ProductPage = ({
               >
                 {loadingAddToCheckout ? "Adding..." : "Add to cart"}
               </button>
-            ) : (
+            ) : null}
+
+            {!selectedVariant && (
+              <p className="text-lg- text-yellow-600">
+                Please choose a variant
+              </p>
+            )}
+
+            {selectedVariant?.quantityAvailable === 0 && (
               <p className="text-lg- text-yellow-600">Sold out!</p>
             )}
+
             {!!addToCartError && <p>{addToCartError}</p>}
+
+            {product?.description && (
+              <div className="text-base text-gray-700 space-y-6">
+                <RichText jsonStringData={product.description} />
+              </div>
+            )}
+
+            {attributes.length > 0 && (
+              <div>
+                <p className="text-lg mt-2 font-medium text-gray-500">
+                  Attributes
+                </p>
+                <div className="">
```

Comment:
```suggestion
                <div className="">
```
If there is no classes, attribute should be omitted.